themes/bootstrap3/theme.config.php

```diff
@@ -27,6 +27,18 @@
     'favicon' => 'vufind-favicon.ico',
     'helpers' => [
         'factories' => [
+            'Zend\Form\View\Helper\Form' => 'Zend\ServiceManager\Factory\InvokableFactory',
```

Comment:
Can you avoid this extra configuration by activating the `Zend\Form` module here: https://github.com/vufind-org/vufind/blob/master/config/application.config.php#L5 ?

Reply:
Yes, much better this way.